src/dymoprint/gui/gui.py

```diff
@@ -246,13 +246,14 @@ def parse(app):
     parser.process(app)
 
     is_verbose = parser.isSet(verbose_option)
-    if is_verbose:
-        set_verbose()
+    if (not is_verbose) and (not is_verbose_env_vars()):
+        # Neither the --verbose flag nor the environment variable is set.
+        set_not_verbose()
 
 
 def main():
+    configure_logging()
```

Comment:
This change better go in https://github.com/tomers/dymoprint/pull/6/commits/001a130d5aa723c881dad906e48ea912ee757b7e, but that's not critical